packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-card-banner.tsx

```diff
@@ -149,8 +147,8 @@ const VerificationStatusBanner = ({
             </div>
         );
     } else if (
-        ((is_bvi || is_labuan) && poi_acknowledged_for_vanuatu_maltainvest && poa_not_submitted) ||
-        ((is_vanuatu || is_maltainvest) && poi_acknowledged_for_bvi_labuan && poa_not_submitted)
+        ((is_bvi || is_labuan || is_vanuatu) && poi_acknowledged_for_bvi_labuan_vanuatu && poa_not_submitted) ||
```

Comment:
since (is_bvi || is_labuan || is_vanuatu) is used multiple times, can we create a varible for it and reuse it?

Reply:
Sure I'll update it. Thanks